tests/test_triton_server.py

```diff
@@ -132,13 +137,28 @@ def test_create_server(self):
                 msg="Expected AssertionError for trying to create"
                 "server without specifying model repository."):
             self.server = TritonServerFactory.create_server_local(
-                path=TRITON_LOCAL_BIN_PATH, config=server_config, gpus=['all'])
+                path=TRITON_LOCAL_BIN_PATH, config=server_config, gpus=gpus)
+
+    def test_create_server_no_gpu(self):
+        self.test_create_server(gpus=[])
+
+    def test_create_server_select_gpu(self):
+        self.test_create_server(
+            gpus=self.gpu_device_factory_mock.get_cuda_visible_gpus()[:1])
+
+    def test_start_stop_gpus(self, gpus=None):
```

Comment:
Same

Reply:
Also, I think the function name can be updated as well.

Reply:
Updated.